### ISSUE TITLE
crpn_faster_rcnn gets "No gt bboxes" error

### ISSUE THREAD

**Issue by reo911gt3** [open-mmlab/mmdetection#4517]
Hello, 

I have been trying to run crpn_faster_rcnn but the training always fails with "No gt bboxes" error. 

I am using custom models but the model does have a region proposals as I am loading the weights from pre-trained weights for crpn I created from the same dataset. 

I've played around with the learning rate but no chance.

Do you have any hint or advice on what is going on? Thanks.

**Comment by reo911gt3** [comment]
This is the error message
![Screenshot 2021-01-22 at 13 03 07](https://user-images.githubusercontent.com/5387810/105494072-2d5dfb00-5cb2-11eb-9b98-8c2dd0c8bcc2.png)

**Comment by nmxian** [comment]
I have met the same problem.Have you solve it?

> This is the error message
> ![Screenshot 2021-01-22 at 13 03 07](https://user-images.githubusercontent.com/5387810/105494072-2d5dfb00-5cb2-11eb-9b98-8c2dd0c8bcc2.png)

**Comment by ZwwWayne** [comment]
It seems that cascade RPN does not support empty GT images from the [code here](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/bbox/assigners/region_assigner.py#L107). I am not sure whether this has specific motivation for that, so may involve @thangvubk's help to have a look on that. Maybe we can enable similar logic for empty GT in other [assigner](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/bbox/assigners/max_iou_assigner.py#L145).

**Comment by thangvubk** [comment]
Hi. I think just enable logic for empty GT in other assigners to resolve the problem. I will update the code.
